waku/waku.md

```diff
@@ -92,15 +91,28 @@ rate-limits = "[" limit-ip limit-peerid limit-topic "]"
 
 light-node = BIT
 
+pow-requirement-key = 0
+bloom-filter-key = 1
+light-node-key = 2
+confirmations-enabled-key = 3
+rate-limits-key = 4
+topic-interest-key = 5
+
 status = "[" 
-        version pow-requirement 
-        [ bloom-filter ] [ light-node ] 
-        [ confirmations-enabled ] [ rate-limits ]
+        version
+        [ pow-requirement-key pow-requirement ]
+        [ bloom-filter-key bloom-filter ]
+	[ light-node-key light-node ] 
+        [ confirmations-enabled-key confirmations-enabled ]
+	[ rate-limits-key rate-limits ]
+	[ topic-interest-key topic-interest ]
     "]"
 
 ; version is "an integer (as specified in RLP)"
 version = DIGIT
 
+topic-node-enabled = BIT
```

Comment:
Not sure what this is.

Reply:
leftover from https://github.com/vacp2p/specs/pull/95 commit, will remove